docs/pages/admin-guides/infrastructure-as-code/teleport-operator.mdx

```diff
@@ -72,6 +72,17 @@ finalizer or remove the ignore annotation.
 
 Possible values are `"true"` or `"false"` (those are strings, as Booleans are not valid label values in Kubernetes).
 
+### Lookup values from secrets
+
+Some Teleport resources might contain sensitive values. Since 16.4, select CR fields can reference an existing
```

Comment:
Nit: I don't think we need the "since 16.4" note here.
```suggestion
Some Teleport resources might contain sensitive values. Select CR fields can reference an existing
```